src/app/shared/mydspace-actions/workspaceitem/workspaceitem-actions.component.ts

```diff
@@ -37,6 +41,20 @@ export class WorkspaceitemActionsComponent extends MyDSpaceActionsComponent<Work
    */
   public processingDelete$ = new BehaviorSubject<boolean>(false);
 
+  /**
+   * A boolean representing if the user is an admin
+   * @type {Observable<boolean>}
+   */
+  isAdmin$: Observable<boolean>;
```

Comment:
This can be removed as it's no longer used.

Reply:
Hello @tdonohue,
Required changes are done